entproto/internal/entprototest/adapter_test.go

```diff
@@ -94,6 +94,14 @@ func (suite *AdapterTestSuite) TestImplicitSkippedMessage() {
 	suite.EqualError(err, entproto.ErrSchemaSkipped.Error())
 }
 
+func (suite *AdapterTestSuite) TestMessageWithStrings() {
```

Comment:
Do you mind changing the receiver to be a single character? In this PR or a different one. Thanks 🙏

Reply:
Sure. I want to deprecate the usage of suite anyways